### PR TITLE
changed accounts_id to required

### PR DESCRIPTION
**Pull request by jiropardo**
## Changes

This pull request changes the (optional) to (required) in the index.md main doc https://registry.terraform.io/providers/databricks/databricks/latest/docs

This is for fixing below doc issue
#3143

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -209,7 +209,7 @@ Alternatively, you can provide this value as an environment variable `DATABRICKS
...
 `DEFAULT`.
-* `account_id` - (optional) Account Id that could be found in the top right corner of [Accounts Console](https://accounts.cloud.databricks.com/). Alternatively, you can provide this value as an environment variable `DATABRICKS_ACCOUNT_ID`. Only has effect when `host = "https://accounts.cloud.databricks.com/"`, and is currently used to provision account admins via [databricks_user](resources/user.md). In the future releases of the provider this property will also be used specify account for `databricks_mws_*` resources as well.
+* `account_id` - (required) Account Id that could be found in the top right corner of [Accounts Console](https://accounts.cloud.databricks.com/). Alternatively, you can provide this value as an environment variable `DATABRICKS_ACCOUNT_ID`. Only has effect when `host = "https://accounts.cloud.databricks.com/"`, and is currently used to provision account admins via [databricks_user](resources/user.md). In the future releases of the provider this property will also be used specify account for `databricks_mws_*` resources as well.
```
Comment (alexott) [inline docs/index.md:212]: It's optional for workspace-level operations, but required for account-level...

Comment (tanmay-db) [reply docs/index.md:212]: Hi @jiropardo, can you please clarify this as @alexott mentioned. 

> (required, for account-level provider)

Comment (jiropardo) [reply docs/index.md:212]: Hello, @alexott

Comment (tanmay-db) [reply docs/index.md:212]: Hi team, following up on this thread, I think this should be good, 

> "It's optional for workspace-level operations, but required for account-level" .

Shall we go ahead with this? cc: @alexott @jiropardo

**Review by alexott — APPROVED** [review]

**Pull request merged**